snakemake/settings/types.py

```
@@ -3,7 +3,7 @@
 import os
 from pathlib import Path
 import re
-from typing import Any, Optional
+from typing import Any, List, Optional, Union
```

Ruff

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)